dev_tools/containers/build_all_containers.sh

```diff
@@ -3,6 +3,8 @@
 
 hpccm --recipe ginkgo-nocuda-base.py --userarg gnu=8 llvm=7 papi=True > gko-nocuda-gnu8-llvm70.baseimage
 list=('gko-nocuda-gnu8-llvm70.baseimage')
+hpccm --recipe ginkgo-nocuda-base.py --userarg gnu=9 llvm=8 papi=True > gko-nocuda-gnu9-llvm8.baseimage
+list+=('gko-nocuda-gnu9-llvm8.baseimage')
```

Comment:
Could the image name be gko-nocuda-gnu9-llvm80.baseimage?

Reply:
In fact, similarly to the previous comment, this is linked to how `llvm` changed their version numbering. Before it was 6.0 (etc), but since `7` it is plain 7. The current numbering for `7` is wrong (we use `7.0` but it should be `7`). but starting with `8` I decided to stick to their new scheme.

You can compare the number given to `llvm`, for numbers older than 7 it's in the form `x.y`.

Reply:
I see.